scripts/dataload.sh

```
@@ -0,0 +1,82 @@
+        done
+        if ! pg_isready -h "$PG_HOST" >/dev/null 2>&1; then
+            pg_isready -h "$PG_HOST"
+            exit $?
+        fi
+        for script_path in "$2"/*; do  # Fix: dùng $2 thay $1, và /* thay ls
+            if [ ! -f "$script_path" ]; then
+                continue
+            fi
+            if ! PGPASSWORD="${PG_PASSWORD:-rootroot}" psql -h "${PG_HOST:-localhost}" \
+                    -U "${PG_USERNAME:-postgres}" \  # Thường là 'postgres' cho PG, không phải 'root'
+                    -d "${PG_DATABASE:-test}" \
```

Shellcheck (0.11.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)